Code/Editor/EditorViewportWidget.cpp

```diff
@@ -1957,6 +1979,24 @@ void EditorViewportWidget::SetDefaultCamera()
     PostCameraSet();
 }
 
+
+//////////////////////////////////////////////////////////////////////////
+void EditorViewportWidget::SetDefaultCameraNearFar()
+{
+    auto m = m_defaultView->GetViewToClipMatrix();
```

Comment:
update: Please use meaningful variable names - here instead of `m` we should use `viewToClip`